modules/client/src/test/java/org/apache/ignite/client/fakes/FakeIgniteTables.java

```diff
@@ -246,7 +246,7 @@ public CompletableFuture<Integer> schemaVersionAtCurrentTime(int tableId) {
     }
 
     private static LockManager lockManager() {
-        HeapLockManager lockManager = new HeapLockManager();
+        HeapLockManager lockManager = new HeapLockManager(1024, 1024);
```

Comment:
What's the point in decreasing slots size here?

Reply:
Because it is not needed to make it more than 1K slots. Just to save the memory.